test/test_ops.py

```diff
@@ -17,13 +18,26 @@
 from torch.testing._internal.jit_metaprogramming_utils import create_script_fn, create_traced_fn, \
     check_alias_annotation
 from torch.testing._internal.jit_utils import disable_autodiff_subgraph_inlining
-from collections.abc import Sequence
+import torch.testing._internal.opinfo_helper as opinfo_helper
 
 
 # Tests that apply to all operators
 class TestOpInfo(TestCase):
     exact_dtype = True
 
+    @classmethod
+    def tearDownClass(cls):
```

Comment:
Thinking more about this... will this correctly call the superclass's tearDownClass?